src/timedate/timedated.c

```diff
@@ -531,9 +531,6 @@ static int method_set_time(sd_bus_message *m, void *userdata, sd_bus_error *erro
         assert(m);
         assert(c);
 
-        return sd_bus_error_setf(error, SD_BUS_ERROR_NOT_SUPPORTED,
-            "Changing system settings via systemd is not supported on NixOS.");
-
```

Comment:
If everybody agree on this (reason see https://github.com/NixOS/systemd/pull/10/commits/fbce0ee69359ad905590a68be0411b5593de24ef) I will squash it into https://github.com/NixOS/systemd/pull/10/commits/ce02fb7ca6585834522825db14c47588e76165a2